src/useRendersCount/__docs__/example.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from 'react';
+import { useRendersCount } from '../useRendersCount';
+
+export const Example: React.FC = () => {
+  const [count, setCount] = useState(0);
```

Comment:
Better to use `useRerender` hook.

In our examples we're trying to reuse our hooks as much as possible.